<paste log output from Kubernetes-mediated prediction, here>

Error from server (AlreadyExists): pods "udacity-ml-microservice" already exists
NAME                      READY   STATUS    RESTARTS   AGE
udacity-ml-microservice   1/1     Running   0          46m
Forwarding from 127.0.0.1:8000 -> 80
Forwarding from [::1]:8000 -> 80
Handling connection for 8000

<output of executing make_predictions.sh>

Port: 8000
{
  "prediction": [
    20.35373177134412
  ]
}